src/Controller/OpenIDConnectController.php

```diff
@@ -106,11 +106,14 @@ class OpenIDConnectController implements ContainerInjectionInterface {
   /**
    * Constructor.
    */
-  public function __construct(AuthProviderService $authProviderService, RequestStack $requestStack, SessionInterface $session, CacheItemPoolInterface $cacheItemPool, LanguageManagerInterface $languageManager, LoggerInterface $logger, RendererInterface $renderer) {
-    $plugin = $authProviderService->getPluginInstance('OpenIDConnect');
-    assert($plugin instanceof OpenIDConnect);
-    $this->plugin = $plugin;
-
+  public function __construct(
+    readonly private AuthProviderService $authProviderService,
+    RequestStack $requestStack,
+  SessionInterface $session,
+  CacheItemPoolInterface $cacheItemPool,
+  LanguageManagerInterface $languageManager,
+  LoggerInterface $logger,
+  RendererInterface $renderer) {
```

Comment:
For consistency, please rewrite to https://stitcher.io/blog/constructor-promotion-in-php-8 when possible.

Reply:
Updated.